swagger/swagger.yml

```diff
@@ -0,0 +1,68 @@
+consumes:
+- application/json
+definitions:
+  account:
+    type: object
+    required: 
+      - user_name
+    properties:
+      user_id:
+        type: string
```

Comment:
missing title, description, and example for all fields.

Reply:
Now added, please check